packages/backend-core/src/events/publishers/backup.ts

```diff
@@ -10,3 +15,12 @@ export async function appBackupRestored(backup: AppBackup) {
 
   await publishEvent(Event.APP_BACKUP_RESTORED, properties)
 }
+
+export async function appBackupTriggered(appId: string, name?: string) {
+  const properties: AppBackupTriggeredEvent = {
+    appId: appId,
+    backupName: name,
+    backupCreatedAt: new Date().toISOString(),
```

Comment:
Posthog will already have a time stamp for this. If we wanted to add our own timestamp I'd suggest adding it to all events in the common code where tenancy, identity etc is added and add a new field to the base when type

Reply:
Makes sense, will remove it - in the case of the import it is the time the backup that is being imported was created so its different anyway!